patches/chromium/cherry-pick-814a27f8522b.patch

```diff
@@ -0,0 +1,63 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Andrey Kosyakov <caseq@chromium.org>
+Date: Fri, 28 Aug 2020 18:55:17 +0000
+Subject: Delegate TargetHandler::Session permission checks to the root client
+
+Bug: 1114636
+Change-Id: Iba3865206d7e80b363ec69180ac05e20b56aade2
+Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2380855
+Reviewed-by: Dmitry Gozman <dgozman@chromium.org>
+Reviewed-by: Devlin <rdevlin.cronin@chromium.org>
+Commit-Queue: Andrey Kosyakov <caseq@chromium.org>
+Cr-Commit-Position: refs/heads/master@{#802736}
+(cherry picked from commit 814a27f8522b6ccddcce1a8f6a3b8fb37128ecf9)
+
+diff --git a/content/browser/devtools/protocol/target_handler.cc b/content/browser/devtools/protocol/target_handler.cc
+index 7a30ab12a1a9fff313e8ea14f5c3bf7a327e6cc1..c43f1c60773bd5279c8f87e1c6474cafb8782b83 100644
+--- a/content/browser/devtools/protocol/target_handler.cc
++++ b/content/browser/devtools/protocol/target_handler.cc
+@@ -369,7 +369,17 @@ class TargetHandler::Session : public DevToolsAgentHostClient {
+                                const std::string& message) override {
+     DCHECK(agent_host == agent_host_.get());
+     if (flatten_protocol_) {
++<<<<<<< HEAD
+       handler_->root_session_->SendMessageFromChildSession(id_, message);
++=======
++      // TODO(johannes): It's not clear that this check is useful, but
++      // a similar check has been in the code ever since the flattened protocol
++      // was introduced. Try a DCHECK instead and possibly remove the check.
++      if (!handler_->root_session_->HasChildSession(id_))
++        return;
++      GetRootClient()->DispatchProtocolMessage(
++          handler_->root_session_->GetAgentHost(), message);
++>>>>>>> 814a27f8522b... Delegate TargetHandler::Session permission checks to the root client
```

Comment:
🤔

Reply:
This part of the patch does not apply at all, I had to remove it. Does the change still make sense?